src/components/game-window/GameWindow.js

```diff
@@ -18,6 +19,12 @@ function GameWindow() {
   }, []);
 
   useEffect(() => {
+    compareCoices();
+    // eslint-disable-next-line
```

Comment:
should be //`eslint-disable-next-line react-hooks/exhaustive-deps` to understand why eslint is disabled for that line.

Reply:
Added